db/maintenance/list.go

```diff
@@ -0,0 +1,56 @@
+package maintenance
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/olekukonko/tablewriter"
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
+	"gopkg.in/errgo.v1"
+
+	"github.com/Scalingo/cli/config"
+	"github.com/Scalingo/cli/io"
+	"github.com/Scalingo/cli/utils"
+	"github.com/Scalingo/go-scalingo/v6"
+)
+
+func List(ctx context.Context, app string, addonName string, paginationOpts scalingo.PaginationOpts) error {
+	caser := cases.Title(language.English)
+	c, err := config.ScalingoClient(ctx)
+	if err != nil {
+		return errgo.Notef(err, "fail to get Scalingo client")
```

Comment:
best practice: we changed our convention for error messages (https://www.notion.so/scalingooriginal/Go-Error-Management-1ca42114c730415e95755027935eaccc?pvs=4#a225d10a1bc544d0b743ed3d5729dae0). We no longer start messages with "fail".
```suggestion
		return errgo.Notef(err, "get Scalingo client")
```